linux_os/guide/system/software/sap/accounts_authorized_local_users_sidadm_orasid.rule

```diff
@@ -42,13 +41,20 @@ ocil: |-
 
 warnings:
     - general: |-
-        Currently this rule only works when there is no or only one SAP System ID exist on
-        on each operating system or virtual machine. The SAP system users <tt>sidadm</tt>,
-        <tt>orasid</tt>, <tt>sapadm</tt> and <tt>oracle</tt> can be automatically
-        detected. If there are multiple SAP systems installed on the operating system
-        or virtual machine, the OVAL test will fail even when every SAP system users is
-        authorized to the system. 
+        Currently this rule only works with following limitations:
         <br /> <br />
-        In this case, please use the general purpose test <tt>accounts_authorized_local_users</tt>
+        1. <tt>SAP system mount directory</tt> is <tt>/sapmnt</tt> (mounted or local file system
+        or a symbolic link to the target directory);
+        <br /> <br />
+        2. there is maximum one SAP System on each operating system or virtual machine (maximum
+        one SID in /sapmnt and /usr/sap).
+        <br /> <br />
```

Comment:
I like how you split it, I would maybe get rid of this newlines - the explanation below is directly connected to the point 2, so it make sense to have it in the same paragraph. Rest is ok.

Reply:
OK. Thanks @dahaic